app/addons/documents/routes-documents.js

```diff
@@ -98,12 +98,17 @@ function(app, FauxtonAPI, BaseRoute, Documents, Changes, Index, DocEditor, Datab
       this.addSidebar();
     },
 
+    getAllDatabases: function () {
```

Comment:
Mind adding in a comment in the code itself saying why it's done - I'd look at it and think it was unnecessary.